internal/bminventory/inventory.go

```diff
@@ -4237,6 +4237,10 @@ func (b *bareMetalInventory) uploadLogs(ctx context.Context, params installer.Up
 		if err != nil {
 			return err
 		}
+		// TODO: add params.HostID when adding log URL to the agent CRD
```

Comment:
Is the TODO still relevant?

Reply:
There were discussions about adding host specific url to agent CRDs, but this is a bit out of scope for this PR.
So added this to explain why I send nil for host ID.
Can either change to a comment or remove if that fits better.

Reply:
It's an event related directly for the host, i think that you should add host ID as well. 
The controller will not do anything with this information but in the cloud users will see which host uploaded logs so i think it will help.

Reply:
Will do.

Reply:
done